app/web_conn.go

```diff
@@ -749,6 +749,11 @@ func (wc *WebConn) shouldSendEvent(msg *model.WebSocketEvent) bool {
 		return wc.GetConnectionID() == msg.GetBroadcast().ConnectionId
 	}
 
+	// if the connection is omitted don't send the message
```

Comment:
I think we need to document the priority order somewhere now that we have a long list of filters.

Something in the lines of:
- In general `allow` overrides `deny`.
- And we move from more specific to less specific.
So the order becomes:
1. ConnectionId.
2. OmitConnectionId.
3. UserId.
4. OmitUserIds.
5. ChannelId.
6. TeamId.

I realize `model.NewWebsocketEvent` does not return any error. But we need to enforce the logic somehow. E.g. if you set UserId, and OmitUserIds both, that's clearly an error. Similarly, if someone sets ChannelId, and OmitConnectionId and expects that it will send to all channel members except that connectionId, that is also a false assumption.